Doc/reference/compound_stmts.rst

```diff
@@ -841,7 +841,7 @@ A literal pattern corresponds to most
                   : | "None"
                   : | "True"
                   : | "False"
-                  : | `signed_number`: NUMBER | "-" NUMBER
+   signed_number: NUMBER | "-" NUMBER
```

Comment:
```suggestion
   signed_number: ["-"] NUMBER
```
?

Reply:
Yeah I did notice that `["-"] NUMBER` would be cleaner but I did not change it because I feel that it is not related to the actual bug. What are others opinion on this--should I make this change?

Reply:
All right I think it over, `["-"] NUMBER` looks better. Thanks for your review.